README.md

```diff
@@ -166,6 +166,39 @@ this will map to the following `vrrp_instance` section
     }
 ```
 
+## Advanced Users Only: Override Keepalived Configuration Template
+
+Create a ConfigMap with the full contents of this configuration template file:
+https://github.com/redhat-cop/keepalived-operator/blob/29402297ca8a252be77904c3f3c29b08e50a3b85/config/templates/keepalived-template.yaml#L166
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+name: keepalived-template
+namespace: {{ .KeepalivedGroup.ObjectMeta.Namespace }}
+labels:
+  keepalivedGroup: {{ .KeepalivedGroup.ObjectMeta.Name }}    
+data: 
+keepalived.conf: |
+    ...
+    # expected merge structure
+    # .KeepAlivedGroup
+    # .Services
+    - apiVersion: apps/v1
+      kind: DaemonSet
+      metadata:
+        name: {{ .KeepalivedGroup.ObjectMeta.Name }}
+        namespace: {{ .KeepalivedGroup.ObjectMeta.Namespace }}
+      spec:
+    ...
+```
+  
+Then in the Helm Chart set `keepalivedTemplateFromConfigMap: keepalived-template`
+
+This will override the `/templates/keepalived-template.yaml` config file in the keepalived-operator pod which will allow you to update the configs without having to rebuild/push the operator docker image.
```

Comment:
can we add a blurb saying that this works only for the helm chart deployment.

Reply:
👍